molecule/test1/molecule.yml

```diff
@@ -4,8 +4,8 @@ driver:
   name: docker
 platforms:
   - name: testohpc-login-0
-    image: docker.io/pycontribs/centos:7
-    pre_build_image: true
+    image: ${MOLECULE_IMAGE}
+    #pre_build_image: true
```

Comment:
Nit: remove the commented configuration?

Reply:
TODO: actually add this back in, centos7 tests running, centos8 tests todo.

Reply:
Added back in for all tests in `2c9b6db`